### ISSUE TITLE
Directive use (ts-mongo) failing when using import feature

### ISSUE THREAD

**Issue by amiiit** [dotansimha/graphql-code-generator#1249]
**Describe the bug**
When using directives such as `@entity` together with `# import` I get the following error:
```
 Directive entity: Couldn't find type entity in any of the schemas
```

**To Reproduce**
I've created the following repo to reproduce the file: https://github.com/amiiit/graphql-directive-import-example

```
git clone git@github.com:amiiit/graphql-directive-import-example.git
cd graphql-directive-import-example
yarn
yarn codegen-with-import # failing case
yarn codegen-no-import # success case
```
1. Use `# import Foo from foo.graphql`
2. In `foo.graphql` have a `type Foo @entity {...}`

**Expected behavior**
Should create types for the mongodb entities

**Schema/Documents**
[The schema using the import statement](https://github.com/amiiit/graphql-directive-import-example/blob/master/schema-import.graphql)
[The imported part of the schema that uses the `@entity` directive](https://github.com/amiiit/graphql-directive-import-example/blob/master/schema-foo.graphql)

**Environment:**
 - OS: Mac OS Sierra 10.12.6
 - Codegen: 0.16.0
 - Node: v10.12.0

**Comment by dotansimha** [comment]
I think it was fixed in the recent refactor, also we fixed something related to it in `graphql-toolkit`.

**Comment by dotansimha** [comment]
Fix available in 1.0.0 👍
